app/lib/actions/pulp/repository/presenters/docker_presenter.rb

```diff
@@ -31,7 +31,7 @@ def humanized_details
 
             if content_started?(download_details)
               if items_total(download_details) > 0
-                ret << (_("New images: %{count}.") % {:count => count_summary})
+                ret << (_("New blobs: %{count}.") % {:count => count_summary})
```

Comment:
@jlsherrill, after further testing, it looks like the format that pulp is generating for the progress is directly related to how it is outputing it to the user with pulp-admin.  Since every repo type from pulp is handling the progress differently, I will attempt to update the docker presenter to accommodate pulp's current format.  I will do that separately using : http://projects.theforeman.org/issues/13480 . 

As a reference, below is output from pulp-admin for a sync:
```
$ pulp-admin docker repo sync run --repo-id=elasticsearch
+----------------------------------------------------------------------+
                Synchronizing Repository [elasticsearch]
+----------------------------------------------------------------------+

This command may be exited via ctrl+c without affecting the request.


Downloading manifests
[/]
... completed

Copying units already in pulp
[\]
... completed

Copying units already in pulp
[-]
... completed

Downloading remote files
[                                                  ] 0%
1 of 209 items
... completed

Saving Manifests and Blobs
[|]
... completed

Saving Tags
[-]
... completed


Task Succeeded

```